src/dags/openbareverlichting.py

```diff
@@ -38,16 +38,17 @@
 total_checks = []
 count_checks = []
 geo_checks = []
-check_name = {}
+
 
 # needed to put quotes on elements in geotypes for SQL_CHECK_GEO
-def quote(instr):
+def quote(instr: str) -> str:
```

Comment:
Not part of this PR, but any future SQL manipulation should be done using https://www.psycopg.org/docs/sql.html

Reply:
👍🏼